frontend/src/components/common/datepicker/datepicker.tsx

```diff
@@ -0,0 +1,51 @@
+import 'react-datepicker/dist/react-datepicker.css';
+
+import {
+  FC,
+  FormControl,
+  FormControlErrors,
+  FormControlPath,
+} from 'common/types/types';
+import { ErrorMessage } from 'components/common/common';
+import { useFormControl } from 'hooks/hooks';
+import DatePicker from 'react-datepicker';
+
+import styles from './styles.module.scss';
+
+type Props = {
+  control: FormControl;
+  errors: FormControlErrors;
+  name: FormControlPath;
+  label: string;
+};
+
+const Datepicker: FC<Props> = ({ control, name, label, errors }) => {
+  const { field } = useFormControl({ name, control });
+
+  const handleChange = (date: Date | null): void => {
+    field.onChange(date);
+  };
+
+  const getDayClassName = (): string => styles.datePickerDay;
```

Comment:
```suggestion
  const handleDayClassNameGet = (): string => styles.datePickerDay;
```

Reply:
Why do we need this helper?

Reply:
@Liza-Veis it's the only way to set classname for day cell, it's required in DatePicker docs.